src/Data/Nat/Properties.agda

```diff
@@ -1617,6 +1621,26 @@ module ≤-Reasoning where
     public
     hiding (_≈⟨_⟩_)
 
+
+open ≤-Reasoning using () renaming (begin_ to begin≤_; _∎ to _end≤;
+                                    _≡⟨_⟩_ to _≡≤[_]_; _≤⟨_⟩_ to _≤[_]_)
```

Comment:
Please could you use the existing names.

Reply:
There is  ``open ≡-Reasoning`` in the beginning of the file.
Much lower it is set ``module ≤-Reasoning where ...``.
And then goes the new code for ``m<m*n``.
It uses ``open ≤-Reasoning``. But all the names in the two "Reasoning" modules will clash, except  only ``_≤⟨_⟩_``. 
?